modules/RsPaymentTerminal/manifest.yaml

```
@@ -40,6 +40,18 @@ config:
   password:
     description: Password for the Feig terminal.
     type: integer
+  accept_credit_cards:
+    description: Indicates whether credit cards should be accepted for all connectors
```

The description "for all connectors" is in contradiction to the description for `credit_card_connectors`, where it says "only [...] to the given connectors".
```suggestion
    description: Indicates whether credit cards should be accepted for the connectors given with `credit_card_connectors`
```